packages/teams-js/src/internal/communication.ts

```diff
@@ -90,7 +90,7 @@ export function initializeCommunication(validMessageOrigins: string[] | undefine
     // Send the initialized message to any origin, because at this point we most likely don't know the origin
     // of the parent window, and this message contains no data that could pose a security risk.
     Communication.parentOrigin = '*';
-    return sendMessageToParentAsync<[FrameContexts, string, string, string]>('initialize', [
+    return sendMessageToParentAsyncWithVersion<[FrameContexts, string, string, string]>('v2', 'initialize', [
```

Comment:
Why is this v2? Is there a v1 veresion of this being passed from somewhere? Was surprised to see this as v2

Reply:
@TrevorJoelHarris , the goal of this PR is to differentiate the versions of API by giving them 'apiVersion' labels. And remember very early ago we said that
```
No matter what API  it is, if it is used in Teams-Js library V1.X.Y, or in the other word, under Teams-Js library V2.0.0, it should be labeled as V1
```

'Initialize' is a funcName argument used in SDK request to help communicating between SDKs and Teams-Js library.

Here is my thought and please correct me because I am probably wrong. 'Initialize' represents API 'initialize' (public API) in Teams-Js library V1.X.Y or API 'App.Initialize' in Teams-Js library ^V2.0.0(V2.X.Y). So, we will assign 'V1' to 'Initialize' API as a public API, and thus code here, which is to send out request for 'App.Initialize', is assigned as 'V2'.